docs/resources/env_azure.md

```diff
@@ -10,7 +9,68 @@ description: |-
 
 Bring Your Own Cloud (BYOC) Azure environment resource.
 
-
+## Example Usage
+
+Azure environment with public Load Balancer:
+```terraform
+provider "azurerm" {
+  skip_provider_registration = true
+  features {}
+}
+
+variable "subscription_id" {
+  type        = string
+  description = "The ID of subscription to connect to Altinity.Cloud"
+}
+
+variable "tenant_id" {
+  type        = string
+  description = "The ID of tenant to connect to Altinity.Cloud"
+}
```

Comment:
none of the other examples we have use variables. I suggest we remove these just to be consistent.